cmd/sonarExecuteScan.go

```diff
@@ -131,6 +144,38 @@ func runSonar(config sonarExecuteScanOptions, client piperhttp.Downloader, runne
 	return nil
 }
 
+func addJacocoReportPaths() {
+	matches, err := glob("**/target/**/*.exec")
```

Comment:
SonarQube discontinued the usage of `jacoco.exec` files.

Reply:
Yes, thanks! I just found out by reading the logs carefully and will change this for the new property (sonar.coverage.jacoco.xmlReportPaths).